tests/test_condaforge_config_schema.py

```diff
@@ -0,0 +1,179 @@
+import pytest
+from pydantic import ValidationError
+from conda_smithy.schema.models import ConfigModel
+
+
+@pytest.fixture
+def default_config_data():
```

Comment:
Are maintainers expected to keep this synced with the code or is this only for the transition period?

Reply:
Just for now, to keep this as a test suit, I wanted to ensure the overall execution of the tests and methods would work. This can be removed as per the other tests we have with the builds -- though we should have some specific ones just for the models, not sure we would go with that though

Reply:
removed